node/pkg/accountant/accountant.go

```diff
@@ -41,26 +41,27 @@ type (
 		BroadcastTxResponseToString(txResp *sdktx.BroadcastTxResponse) string
 	}
 
-	// tokenBridgeKey is the key to the map of token bridges being monitored
-	tokenBridgeKey struct {
+	// emitterKey is the key to a map of emitters to be monitored
+	emitterKey struct {
 		emitterChainId vaa.ChainID
 		emitterAddr    vaa.Address
 	}
 
-	// tokenBridgeEntry is the payload of the map of the token bridges being monitored
-	tokenBridgeEntry struct {
-	}
+	// validEmitters is a set of supported emitter chain / address pairs. The payload is the enforcement flag.
+	validEmitters map[emitterKey]bool
```

Comment:
I'm not expecting a name change here.  Just getting my bearings. Isn't this really *enforcedEmitters*?  Any emitter in a map would just assumed to be valid?

Reply:
No, the map would contain all valid emitters, but some of them may not have enforcement enabled ("log only"). So this map really does contain all valid emitters.